src/lsptoolshost/builtInComponents.ts

```diff
@@ -27,9 +27,14 @@ export const componentInfo: { [key: string]: ComponentInfo } = {
             'Microsoft.VisualStudio.DesignTools.CodeAnalysis.Diagnostics.dll',
         ],
     },
+    razorDevKit: {
+        defaultFolderName: '.razorDevKit',
+        optionName: 'razorDevKit',
+        componentDllPaths: ['Microsoft.VisualStudio.DevKit.Razor.dll'],
+    },
 };
 
-export function getComponentPaths(componentName: string, options: LanguageServerOptions): string[] {
+export function getComponentPaths(componentName: string, options: LanguageServerOptions | undefined): string[] {
```

Comment:
instead of having this undefined you could consider adding the razorDevkit folder as a component path option.  Then you would be able to override it for local development.

Reply:
I think the only issue I have with that is the structure `LanguageServerOptions` has a lot of concepts that don't apply to razor. Could we move the options to a lighter interface?

Reply:
I guess technically the other options aren't relevant to you, but you can just import `languageServerOptions` and pass it to the function (without caring what else is on it).  Actually, frankly `builtInComponents.ts` should probably just call `languageServerOptions.componentPaths` and not even have options be a parameter.

Reply:
We could move it to `CommonOptions` as well, but not sure if its necessary to do that

Reply:
> Actually, frankly builtInComponents.ts should probably just call languageServerOptions.componentPaths and not even have options be a parameter

I'll look into that. Seems cleaner

Reply:
Actually let me see about that in a follow up. Would like to get this in for now since I'll likely have another razor change coming in that will require dual insertion